src/System.Management.Automation/engine/remoting/commands/InvokeCommandCommand.cs

```diff
@@ -263,6 +263,33 @@ public override SwitchParameter UseSSL
             }
         }
 
+        /// <summary>
+        /// Gets or sets strict mode.
+        /// </summary>
+        [Experimental("InvokeCommandCommand.PSStrictModeVersionAssignment", ExperimentAction.Show)]
```

Comment:
This is causing tests to fail.  Experimental name is `PSStrictModeVersionAssigment`, not `InvokeCommandCommand.PSStricteModeVersionAssignment`.
```suggestion
        [Experimental("PSStrictModeVersionAssignment", ExperimentAction.Show)]
```